bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/WemoUtil.java

```diff
@@ -155,4 +156,29 @@ private static boolean servicePing(String host, int port) {
         entities.put("quot", "\"");
         return entities;
     }
+
+    public static @Nullable String createBinaryStateContent(String binaryState) {
```

Comment:
I would propose to let this method take a **boolean** parameter and let it convert to "0" / "1". I don't see any reason not to centralize this logic here as opposed to letting all callers perform this conversion?

Reply:
Done